bin/q

```diff
@@ -641,7 +647,7 @@ class TableColumnInferer(object):
         counts = {}
         for column_count in column_count_list:
             counts[column_count] = counts.get(column_count, 0) + 1
-        return ", ".join(["%s rows with %s columns" % (v, k) for k, v in counts.iteritems()])
+        return six.u(", ").join([six.u("{} rows with {} columns".format(v, k)) for k, v in six.iteritems(counts)])
```

Comment:
six.u() and six.b() and not widely used these daze with u"string" and b"string" being supported on all supported versions of Python.

Reply:
Thanks. Will do a complete sweep over all of those after everything works.